README.md

```diff
@@ -9,101 +9,22 @@
 
 Calcite Web is a web design framework that implements the Esri Brand Guidelines and Calcite design framework for browser-based properties and products. Calcite Web is written in [Sass](http://sass-lang.com/), and is compatible with both the standard Sass ruby gem and the [Libsass](http://libsass.org/) compiler. The project also includes a dependency-free JavaScript library for use with interactive patterns.
 
-# Using Calcite Web
+## Getting Started
 
-There are two main ways to use Calcite Web: copying static files into your project and installing via a package manager. Whatever your flavor, we probably have you covered.
+We recommend installing Calcite Web with a package manager. Calcite Web works as both a [ruby gem](http://esri.github.io/calcite-web/documentation/#ruby-gem) or [NPM module](http://esri.github.io/calcite-web/documentation/#npm). Check out the [documentation](http://esri.github.io/calcite-web/documentation/) for installation instructions.
 
-## Static Files
+Once you've installed Calcite Web, it can be helpful to read the [quickstart guide](http://esri.github.io/calcite-web/guides/quickstart/) to get an idea of how you use it.
 
-This is probably the easiest way. If you're looking to get up and running quickly, just [download the latest release](https://github.com/esri/calcite-web/releases) and move the zipped files to wherever you keep your assets. Be sure to use [the documentation site](http://esri.github.io/calcite-web/) to copy and paste patterns, components, and even a sample html boilerplate.
+## Browser Support
 
-## Ruby Gem
+| <img src="https://raw.githubusercontent.com/godban/browsers-support-badges/master/src/images/edge.png" alt="IE / Edge" width="16px" height="16px" /></br>IE / Edge | <img src="https://raw.githubusercontent.com/godban/browsers-support-badges/master/src/images/firefox.png" alt="Firefox" width="16px" height="16px" /></br>Firefox | <img src="https://raw.githubusercontent.com/godban/browsers-support-badges/master/src/images/chrome.png" alt="Chrome" width="16px" height="16px" /></br>Chrome | <img src="https://raw.githubusercontent.com/godban/browsers-support-badges/master/src/images/safari.png" alt="Safari" width="16px" height="16px" /></br>Safari |
+| -------------- | ------------- | ------------- | ------------- |
+|IE10, IE11, Edge|last 2 versions|last 2 versions|last 2 versions|
```

Comment:
I understand, because you’ve combined them, but still, `Edge` is not a version. Would it be easy to just separate Edge and IE into separate columns?